### PR TITLE
Add project files - DateTime check if workday or weekend day

### PR DESCRIPTION
**Pull request by byby9999**

### CODE REVIEW

File: csharp-advanced-topics/CheckDateTimeIsWeekend/ConsoleApp1/DateTimeWeekdayWeekendCheck.csproj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/CheckDateTimeIsWeekend/ConsoleApp1/DateTimeWeekdayWeekendCheck.csproj:4]: We should use .NET 6.

File: csharp-advanced-topics/CheckDateTimeIsWeekend/DateTimeWeekdayWeekendCheck.Tests/DateTimeWeekdayWeekendCheck.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/CheckDateTimeIsWeekend/DateTimeWeekdayWeekendCheck.Tests/DateTimeWeekdayWeekendCheck.Tests.csproj:7]: We should use .NET 6.

**Comment by CodeMazeBlog** [comment]
You've probably created a PR from the main branch. This way you've committed both the evaluation article source code and the code for this article. You can check that in the files section. Please create a branch, transfer the source code for this article and create a PR from that branch to our main. That way we'll get a clean commit history.

**Comment by byby9999** [comment]
will not merge this, will create separate branch

**Pull request closed without merging**